df.world.xml

```diff
@@ -1120,7 +1120,7 @@
                 <int32_t name='disturbance_interaction_number'/>
                 <int32_t name='evil_cloud_number'/>
                 <int32_t name='evil_rain_number'/>
-                <int32_t since='v0.40.01'/>
+                <int32_t name='generate_divine_materials'/>
```

Comment:
Leaving the `since` attribute would be better (it sounds like it's accurate from the name).